packages/api-evm/source/actions/call.ts

```diff
@@ -15,9 +27,9 @@ export class CallAction implements Contracts.Api.RPC.Action {
 			{
 				additionalProperties: false,
 				properties: {
-					data: { type: "string" },
-					from: { type: "string" },
-					to: { type: "string" },
+					data: { $ref: "0xHex" },
```

Comment:
Can we call it `prefixedHex` so it matches the exported schema name? (or call the other `0xHex`)